packages/qdrant-loader-core/src/qdrant_loader_core/graph/extractor/jira.py

```
@@ -0,0 +1,276 @@
+        for url in self.CONFLUENCE_URL_RE.findall(text):
+            edges.append(
+                GraphEdge(
+                    source=doc.id,
+                    target=url,
+                    edge_type=CoreEdgeType.LINKS_TO,
+                    project=project,
+                    properties={
+                        "kind": "confluence",
+                    },
+                )
+            )
```

Ruff (0.15.15)

[warning] Use a list comprehension to create a transformed list

(PERF401)